packages/util/test/array.spec.ts

```diff
@@ -1,7 +1,35 @@
-import { flat, unique, last, first, splitInChunks } from '../src';
+import {
+  flat,
+  unique,
+  last,
+  first,
+  splitInChunks,
+  variableArgumentToArray
+} from '../src';
 
 describe('array', () => {
   describe('flat', () => {
+    it('wraps variable arguments', () => {
```

Comment:
Why is this in flat?

Reply:
by mistake. I moved it to its own section.